Instructions for Connecting Vertex AI to LLM Observability (#33470)
DataDog/documentation: content/en/llm_observability/evaluations/managed_evaluations/_index.md

[1]: https://app.datadoghq.com/llm/settings/integrations
{{% /tab %}}

{{% tab "GCP Vertex AI" %}}

> <div class="alert alert-info">If you are subject to HIPAA, you are responsible for ensuring that you connect only to a Google Cloud Platform account that is subject to a business associate agreement (BAA) and meets all requirements for HIPAA compliance.</div>

Reviewer: ```suggestion
<div class="alert alert-info">If you are subject to HIPAA, ensure that you only connect to a Google Cloud Platform account covered by a business associate agreement (BAA) that meets all HIPAA compliance requirements.</div>
```
Suggestion to make this a bit more concise.

Connect Vertex AI to LLM Observability with your Google Cloud Platform account. LLM Observability uses the `gemini-2.5-flash` model for evaluations.

1. In Datadog, navigate to [**LLM Observability > Settings > Integrations**][1].
1. Select **Connect** on the Google Cloud Vertex AI tile if you have no GCP accounts already connected to Datadog. If you have GCP accounts already connected, select **Configure** next to where they are listed.
1. Follow the onboarding instructions to configure your account.

>   - You will have to add the [**Vertex AI User**][2] role to your account and enable the [**Vertex AI API**][3].

Reviewer: ```suggestion
  - Add the [**Vertex AI User**][2] role to your account and enable the [**Vertex AI API**][3].
```
